ui/component/viewers/videoViewer/internal/chapters.jsx

```diff
@@ -184,8 +184,8 @@ export function parseAndLoad(player: any, claim: StreamClaim) {
     overrideHoverTooltip(player, tsData, duration);
   } else {
     deleteHoverInformation(player);
-    // $FlowIssue
-    player?.controlBar?.getChild('ChaptersButton')?.hide();
+    // $FlowFixMe
+    player.controlBar.getChild('ChaptersButton').hide();
```

Comment:
Hm ... I think Anthony added `?` to address null controls with some scenarios.
Merge error?